apps/src/aiTutor/views/AssistantMessageFeedback.tsx

```diff
@@ -22,7 +25,17 @@ const AssistantMessageFeedback: React.FC<AssistantMessageProps> = ({
   });
   const [detailsOpen, setDetailsOpen] = useState<boolean>(false);
 
+  const level = useAppSelector(state => state.aiTutor.level);
+
   const handleIconClick = (thumbsUp: boolean, thumbsDown: boolean) => {
+    analyticsReporter.sendEvent(EVENTS.AI_TUTOR_FEEDBACK_SUBMITTED, {
+      levelId: level?.id,
+      levelType: level?.type,
+      progressionType: level?.progressionType,
+      chatMessageId: messageId,
+      thumbsUp: thumbsUp,
+      thumbsDown: thumbsDown,
```

Comment:
I could probably just have one of these and let the boolean speak for itself based on process of elimination, but it felt clearer without really a downside to just have both 👍 and 👎.